<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeDetectionFilterWidget</class>
 <widget class="QWidget" name="EdgeDetectionFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="sensitivityLayout">
     <item>
      <widget class="QLabel" name="sensitivityLabel">
       <property name="text">
        <string>Чувствительность:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableSlider" name="sensitivitySlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sensitivityValueLabel">
       <property name="text">
        <string>0.50</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="operatorLayout">
     <item>
      <widget class="QLabel" name="operatorLabel">
       <property name="text">
        <string>Оператор:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="operatorComboBox">
       <item>
        <property name="text">
         <string>Sobel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prewitt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scharr</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableSlider</class>
   <extends>QSlider</extends>
   <header>gui/ClickableSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

